### PR TITLE
Update free-courses-pt_BR.md

### PR DESCRIPTION
**Pull request by FranciscoHeronildo**
## What does this PR do?
Add resource

### Description
Add resource  about  NoodeJS

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-pt_BR.md [hunk 1]
```diff
@@ -246,6 +246,7 @@
 * [Curso de Node.js](https://www.youtube.com/playlist?list=PLJ_KhUnlXUPtbtLwaxxUxHqvcNQndmI4B) - Victor Lima Guia do Programador (YouTube)
 * [Do Zero A Produção: Aprenda A Construir Uma API Node.Js Com Typescript](https://www.youtube.com/playlist?list=PLz_YTBuxtxt6_Zf1h-qzNsvVt46H8ziKh) - Waldemar Neto Dev Lab (YouTube)
 * [Imersão em desenvolvimento de APIs com Node.js](https://erickwendel.teachable.com/p/node-js-para-iniciantes-nodebr) - Erick Wendel (Teachable)
+* [Node.js - Fundamentos](https://www.treinaweb.com.br/curso/nodejs-fundamentos) - Wesley Gado (TreinaWeb)
```
Comment (shrn01) [inline courses/free-courses-pt_BR.md:249]: Looks like this is a paid course. Please add only courses which are free to use

Comment (FranciscoHeronildo) [reply courses/free-courses-pt_BR.md]: @shrn01 updated course link

**Review by shrn01 — APPROVED** [review]
LGTM

File: courses/free-courses-pt_BR.md [hunk 2]
```diff
@@ -243,6 +243,7 @@
 ### Node.js
 
 * [Criando APIs com NodeJs](https://www.youtube.com/playlist?list=PLHlHvK2lnJndvvycjBqQAbgEDqXxKLoqn) - Balta.io (YouTube)
+* [Curso de Node](https://www.youtube.com/watch?v=XN705pQeoyU&list=PLx4x_zx8csUjFC41ev2qX5dnr-0ThpoXE) - CFBCursos (YouTube)
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:246]: For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Comment (FranciscoHeronildo) [reply courses/free-courses-pt_BR.md:246]: @eshellman This is a link to a playlist on youtube, just access it to view. And as it is presented in the contribution guide I write the name of the youtube channel. Give me another suggestion to improve.

File: courses/free-courses-pt_BR.md [hunk 3]
```diff
@@ -243,6 +243,7 @@
 ### Node.js
 
 * [Criando APIs com NodeJs](https://www.youtube.com/playlist?list=PLHlHvK2lnJndvvycjBqQAbgEDqXxKLoqn) - Balta.io (YouTube)
+* [Curso de Node](https://www.youtube.com/watch?v=XN705pQeoyU&list=PLx4x_zx8csUjFC41ev2qX5dnr-0ThpoXE) - Professor Bruno CFBCursos (YouTube)
```
Comment (eshellman) [inline courses/free-courses-pt_BR.md:246]: we don't include honorifics such as "Prof."

Comment (FranciscoHeronildo) [reply courses/free-courses-pt_BR.md]: @eshellman adjusted

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**